validator/rpc/standard_api.go

```diff
@@ -98,6 +103,20 @@ func (s *Server) ImportKeystores(
 			return &ethpbservice.ImportKeystoresResponse{Statuses: statuses}, nil
 		}
 	}
+	if len(req.Passwords) == 0 {
+		req.Passwords = make([]string, len(req.Keystores))
+	}
+
+	// req.Passwords and req.Keystores are checked for 0 length in code above.
+	if len(req.Passwords) > len(req.Keystores) {
+		req.Passwords = req.Passwords[:len(req.Keystores)]
+	}
+	if len(req.Passwords) < len(req.Keystores) {
```

Comment:
Why isn't this an error condition?

Reply:
what do you mean?

Reply:
What I mean is that when the user sends 2 passwords and 3 keystores, how do you unlock the 3rd keystore?

Reply:
oh it'll just have the keystore unable to unlock in the function below and it will set status as error.